ui/cypress/support/index.js

```diff
@@ -17,6 +17,7 @@
 import './commands'
 
 import 'cypress-pipe'
+import 'cypress-plugin-tab'
```

Comment:
js newb so ignore if not relevant, but is this import used anywhere (i assume the linter would complain if not, and that it could be used in another file)?